src/back/controllers/api/routes/my/my.js

```diff
@@ -14,5 +14,6 @@ function responseFactory(action) {
 module.exports = {
```

Comment:
For now yes, but you will add pagination got getting challenges and maybe filtering. In that case you will need to pas it somehow. Maybe for now it is good enough, but it limits your action a lot.